### PR TITLE
Execute the docker image as the running uid:gid

### PR DESCRIPTION
**Pull request by cgrigis**
I also reduced the debug level as 4 is *really* too verbose ;-)

### CODE REVIEW

File: mk/service/backend.mk [hunk 1]
```diff
@@ -62,9 +62,10 @@ define $Swith-conodes-sh =
 		n=$$(docker run --detach --rm \
 			--env CONODE_SERVICE_PATH=/config \
 			--volume $(CURDIR)/$Dbackend/build/conode-$$i:/config \
+			--user `id -u`:`id -g` \
```
Comment (tharvik) [inline mk/service/backend.mk:65]: Hum, to avoir launching `id` too much which won't change during the run, better use `$$UID:$$GID` instead, that should provide the same feature.

**Review by tharvik — CHANGES_REQUESTED** [review]

**Comment by tharvik** [comment]
Otherwise, LGTM

File: mk/service/backend.mk (hunk 1 above)
Comment (cgrigis) [reply mk/service/backend.mk:65]: It looks like it depends on the shell... bash e.g. defines UID but not GID:
https://www.gnu.org/software/bash/manual/bash.html#Bash-Variables

**Comment by ineiti** [comment]
@tharvik this is your repo - so feel free to merge if you're OK with it.

File: mk/service/backend.mk (hunk 1 above)
Comment (tharvik) [reply mk/service/backend.mk:65]: Ho, right, and POSIX doesn't define any; good for me then

**Pull request merged**